.mkosi/mkosi.opensuse

```diff
@@ -47,8 +47,8 @@ BuildPackages=
         pcre-devel
         python3
         python3-lxml
-        qrencode-devel
```

Comment:
uh, no, let's leave this in, this is for testing stuff, hence we want to test comprehensively

Reply:
I saw this as a not tested fringe feature. OTOH, it's not speeding up the build greatly, so I can return in back.

Reply:
Force pushed.

Reply:
we want to add recovery key support to homed soon, and i figure we'll use qr codes for that too